src/main/java/duckling/Server.java

```diff
@@ -1,11 +1,101 @@
 package duckling;
 
+import duckling.requests.Request;
+import duckling.requests.RequestBuilder;
+import duckling.responders.Responders;
+
+import java.io.*;
+import java.net.*;
+import java.util.concurrent.*;
+
 public class Server {
+    public static final String CLRF = "\r\n";
```

Comment:
A semi-common java pattern is to use a class as basically a "namespace" to put a bunch of things that are static, like the way jUnits `Assert` class has a bunch of static methods.  If more broadly useful constant values come up then it might be worth making something like an `HttpConstants` class to put them in.